Cargo.toml

```diff
@@ -159,7 +160,7 @@ full = [
     # parses timezones used in timestamp conversions
     "chrono-tz",
 ]
-arrow = ["arrow-buffer", "arrow-schema"]
+arrow = ["arrow-buffer", "arrow-schema", "arrow-data"]
```

Comment:
Not per se related to this PR. But could we name this `arrow-rs`? I can imagine `arrow` being too generic here.

Reply:
I don't mind, I was just going off that arrow-rs is called arrow